src/Redux/mission/missionSlice.js

```diff
@@ -0,0 +1,42 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+export const fetchMissions = createAsyncThunk('missions/fetchMissions',
+  async () => {
+    const response = await fetch('https://api.spacexdata.com/v3/missions');
+    const data = await response.json();
+
+    return data;
+  });
+
+const initialState = {
+  missions: [],
+
+};
+const missionSlice = createSlice({
```

Comment:
- [ ] [Optional] #L13 seems to be an [extra blank line](https://github.com/microverseinc/curriculum-html-css/blob/b67a5019377900c4957a0e7c6b56020518ee273f/articles/html_css_best_practices.md?plain=1#L6)
it is a best practice to make sure not to have extra blank lines.

- [ ] [Optional] #L14 -> #L15 a blank line between these should help readability, and avoid missing the closing/opening of the code blocks.